meta/dates.json

```diff
@@ -909,5 +909,6 @@
   "blink-mind-desktop": "2020-04-28",
   "ioui": "2020-04-28",
   "kankan": "2020-04-28",
-  "merge-request-notifier": "2020-04-28"
+  "merge-request-notifier": "2020-04-28",
+  "gdevelop": "2020-04-28"
 }
```

Comment:
Should I add a newline?

Reply:
There is no new line before so no need to add one.

Reply:
ok, I ll add one then

Reply:
I said the opposite :/

Reply:
I am sorry @4ian my exams have really got my head covered up :(. I will revert the change and squash the commits if the project collaborators ask me to do so.